tests/ctl/api/test_seed.py

```diff
@@ -436,3 +444,193 @@ async def test_load_orginizations(loguru_caplog, async_session, monkeypatch):
 
     assert "INSERTED 1" in loguru_caplog.text
     assert f"SKIPPED {current_orgs}" in loguru_caplog.text
+
+
+@pytest.mark.integration
+class TestLoadSamples:
+    """Tests related to load_samples"""
+
+    SAMPLE_ENV_VARS = {
+        # Include test secrets for Postgres and Stripe, only
+        "FIDES_DEPLOY__CONNECTORS__POSTGRES__HOST": "test-var-expansion",
+        "FIDES_DEPLOY__CONNECTORS__POSTGRES__PORT": "9090",
+        "FIDES_DEPLOY__CONNECTORS__POSTGRES__DBNAME": "test-var-db",
+        "FIDES_DEPLOY__CONNECTORS__POSTGRES__USERNAME": "test-var-user",
+        "FIDES_DEPLOY__CONNECTORS__POSTGRES__PASSWORD": "test-var-password",
+        "FIDES_DEPLOY__CONNECTORS__STRIPE__DOMAIN": "test-stripe-domain",
+        "FIDES_DEPLOY__CONNECTORS__STRIPE__API_KEY": "test-stripe-api-key",
+    }
+
+    @patch.dict(os.environ, SAMPLE_ENV_VARS, clear=True)
+    async def test_load_samples(
```

Comment:
In my own personal opinion I'm not a huge fan of mega-asserts in tests. I generally find it easier to debug if tests can only fail for one specific reason. I would prefer a pattern where there is an individual test for each resource type, even if it is done via parametrization

Reply:
I'd like that, but a consequence of the test setup right now is that the database is persisted (at least for ctl tests) so they aren't isolated from each other in a meaningful way... so it'd just add a lot of test time but all pass or fail as a group?